src/ints/mouse.cpp

```diff
@@ -1099,6 +1190,135 @@ Bitu UIR_Handler(void) {
 	return CBRET_NONE;
 }
 
+// ***************************************************************************
+// External notifications
+// ***************************************************************************
+
+void Mouse_SetSensitivity(int32_t sensitivity_x, int32_t sensitivity_y) {
+    static constexpr float MIN = 0.01f;
+    static constexpr float MAX = 100.0f;
+
+    mouse_config.sensitivity_x = std::clamp(sensitivity_x / 100.0f, -MAX, MAX);
+    if (!std::signbit(mouse_config.sensitivity_x))
```

Comment:
What's the benefit of doing this comparison the complicated way? Just comparing it to `0.0f` would be much simpler.